source/guides/maintenance/container-image-modify.rst

```diff
@@ -93,131 +102,103 @@ Dockerfile.
       -cares +digest -gpgme +https +ipv6 -iri +large-file -metalink +nls
       -ntlm +opie -psl +ssl/openssl
 
-Example 2: Change |CL| version (single-stage Dockerfiles)
-=========================================================
+Example 2: Change |CL| version (single-stage build)
+***************************************************
 
 This example shows how to rebuild single-stage containers against a specific
-OS version, :file:`<CL_VERSION>`, by adding a new argument to the Docker build command line.
-
-#. Open a Terminal on |CL| and enter the
-   :file:`dockerfiles/machine-learning-ui` directory.
+OS version, :file:`<CL_VERSION>`, by adding a new argument to the Docker build
+command line. 
 
 #. Rebuild the :file:`clearlinux/machine-learning-ui`. Add an extra build
-   argument :command:`swupd_args="-m <CL_VERSION>"`; in this case, the build version is 31090.
+   argument :command:`swupd_args="-m <CL_VERSION>"`; in this case, the build
```

Comment:
Example 2, Step 1: 
Is it possible you could follow the same pattern in this example as you did in Example 1, Step 2-3? (Ln 50). 
I suppose there are some who know what you mean when you write, "Add and extra build argument :command:`swupd_args="-m 31110"`

I like how explicit the steps are in Example 1.